src/frontends/paddle/tests/test_models/gen_scripts/generate_silu.py

```diff
@@ -0,0 +1,48 @@
+#
+# silu paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import paddle as pdpd
+import sys
+
+
+def silu(name: str, x, data_type):
+    pdpd.enable_static()
+
+    with pdpd.static.program_guard(pdpd.static.Program(), pdpd.static.Program()):
+        node_x = pdpd.static.data(
+            name='input_x', shape=x.shape, dtype=data_type)
+        out = pdpd.nn.functional.silu(x=node_x, name='silu')
+
+        cpu = pdpd.static.cpu_places(1)
+        exe = pdpd.static.Executor(cpu[0])
+        # startup program will call initializer to initialize the parameters.
+        exe.run(pdpd.static.default_startup_program())
+
+        outs = exe.run(
+            feed={'input_x': x},
+            fetch_list=[out])
+
+        saveModel(name, exe, feedkeys=['input_x'], fetchlist=[out],
+                  inputs=[x], outputs=[outs[0]], target_dir=sys.argv[1])
+
+    return outs[0]
+
+
+def main():
```

Comment:
Could you help add some `dynamic shape` and different `data type` test cases?

Reply:
Hi @xczhai, I have added some test cases related to `dynamic shape`. `Silu` currently only supports `float32` and `float64`, but OpenVINO currently does not support `float64`, so only `float32` test cases is available. The new test results are shown in the following screenshot.

![image](https://user-images.githubusercontent.com/32707008/233948895-127140c0-1536-4347-81ad-2e528518b211.png)

Reply:
@FisherWY 
Thanks for your contribution first.
For `float64`, OpenVINO IR `Swish` doesn't support it for now since we find data type `float64` is seldom used in such OP.
Don't add an assertion here and just ignore it. 
If more scenarios take `float64` in the future, we will consider to support this dtype.